rules/sairedis.mk

```diff
@@ -8,40 +8,43 @@ $(LIBSAIREDIS)_DPKG_TARGET = binary-sairedis
 $(LIBSAIREDIS)_SRC_PATH = $(SRC_PATH)/sonic-sairedis
 $(LIBSAIREDIS)_VERSION = $(LIBSAIREDIS_VERSION)
 $(LIBSAIREDIS)_NAME = $(LIBSAIREDIS_NAME)
-$(LIBSAIREDIS)_DEPENDS += $(LIBSWSSCOMMON_DEV)
+$(LIBSAIREDIS)_DEPENDS += $(LIBSWSSCOMMON_DEV) $(LIBGCOV_PRELOAD)
```

Comment:
Not sure my understanding is correct. Is it for code coverage? Is it possible to install it on demand, we can enable it in PR build, for official build to install in devices, looks like it is not necessary.

Reply:
Currently, I have raised a sairedis PR to support GCOV, but need preload lib(the reason why can't build with sairedis repo together has mentioned in PR: https://github.com/sonic-net/sonic-sairedis/pull/972). As I know, pipeline framework will download vs docker and install sairedis deb into it, and then do testing. So i need to make sure that the downloaded docker encompass preload lib. 
 Is it possible to install it on demand -> I think a MACRO "ENABLE_GCOV" defined in rules/config will be fine. I will add it soon, default value set to 'y'.

Reply:
We do not want LIBGCOV_PRELOAD in the sonic image. You only need it in test.

Reply:
Are there other ways to solve this problem?  to make sure that vs docker downloaded by pipeline framework when testing encompass gcov preload lib and this lib will not in sonic image ?

Reply:
> We do not want LIBGCOV_PRELOAD in the sonic image. You only need it in test.

Do you have any idea about how to solve this problem?